applications/faq_tool/launch_faq_tool.py

```
@@ -7,11 +7,11 @@
 from dotenv import load_dotenv
 
 sys.path.append('../../')
-from demos.components.open_router_client import OpenRouterClient, GPT_4O_MINI_1807
+from demos.components.open_router_client import OpenRouterClient, GEMINI_2_FLASH_1
 from demos.tool_calling.tool_descriptors import (tools_rag_descriptor)
 
 # noinspection PyUnresolvedReferences
-from tools_rag import lookup_in_documentation
+from tools_rag import lookup_in_documentation, list_documents
```

Ruff (0.8.2)

`tools_rag.lookup_in_documentation` imported but unused

Remove unused import

(F401)